debian/build-debs

```diff
@@ -1,40 +0,0 @@
-#!/usr/bin/env bash
-
-set -x
-set -e
-
-if [ -z "${DEB_VERSION}" ]; then
-  echo "DEB_VERSION variable should be set"
-  exit 1
-fi
-if [ -z "${STREAM}" ]; then
-  echo "STREAM variable should be set to trusty/xenial/bionic..."
-  exit 1
-fi
-
-deb_dir=dist/$STREAM
-unpacked_src_dir=${deb_dir}/calico-${DEB_VERSION}
-
-rm -rf ${deb_dir}
-mkdir -p ${unpacked_src_dir}/bin
-cp bin/calico-felix ${unpacked_src_dir}/bin
```

Comment:
Where is this in the packaging repo? Seems like this is being removed here but I don't see where it exists there.

Reply:
Good observation!  This is indeed being removed overall - it's what's described by the "build-debs previously prepared ..." paragraph in the PR description above.  To put it in other words: previously, the deleted lines here prepared a parallel source tree which only contained calico-felix and a few other files that we install, and did not contain the real Git source.  Now, instead, we just build calico-felix and add it to the real Git source, before calling dpkg-buildpackage.  Does that make sense?

Reply:
Sorry I should have paid more attention to that paragraph.
You mention the `source` target in debian/rules, though I don't see anything there by that name. 
Is that a magic built in target?

Reply:
Ah, I'm sorry, I changed the source target later.  I did have this in Felix's debian/rules:
```
source:
      make bin/calico-felix
```
and then the script in packaging did `debian/rules source`.

But then I thought that was unnecessarily complex, and it would be simpler for the script in packaging just to do `make bin/calico-felix`.